ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/HttpInputPollerTest.java

```diff
@@ -68,6 +72,36 @@ void whenPeriodicPolling_thenReturnTwoTimesTheSameResponse() {
 		WireMock.verify(2, getRequestedFor(urlEqualTo(ENDPOINT)));
 	}
 
+	@Test
+	void whenPollMultipleEndpoints_andOneEndpointFails_thenTheOtherEndpointShouldStillBePolled() {
+		stubFor(get(ENDPOINT).willReturn(serverError().withHeader("Content-Type", CONTENT_TYPE).withBody(CONTENT)));
+		String otherEndpoint = "/other-resource";
+		stubFor(get(otherEndpoint).willReturn(ok().withHeader("Content-Type", CONTENT_TYPE).withBody(CONTENT)));
+		httpInputPoller = new HttpInputPoller(executor, adapter, List.of(BASE_URL + ENDPOINT, BASE_URL + otherEndpoint),
+				true);
+
+		httpInputPoller.poll();
+
+		WireMock.verify(getRequestedFor(urlEqualTo(ENDPOINT)));
+		WireMock.verify(getRequestedFor(urlEqualTo(otherEndpoint)));
+	}
+
+	@Test
+	void whenPeriodicPollingMultipleEndpoints_thenReturnTwoTimesTheSameResponse() {
+		String endpoint = "/endpoint";
```

Comment:
Is there a reason for not using the static final variable ENDPOINT?

Reply:
I wanted to be explicit that endpoint and otherEndpoint are different